<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="21627" started-at="2019-07-08T06:02:17Z" finished-at="2019-07-08T06:02:39Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="21627" started-at="2019-07-08T06:02:17Z" finished-at="2019-07-08T06:02:39Z">
      <class name="com.jenkins.testforjenkins.Test01Test">
        <test-method status="PASS" signature="startChrome()[pri:0, instance:com.jenkins.testforjenkins.Test01Test@4566e5bd]" name="startChrome" is-config="true" duration-ms="18304" started-at="2019-07-08T14:02:17Z" finished-at="2019-07-08T14:02:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startChrome -->
        <test-method status="FAIL" signature="openBaiDu()[pri:0, instance:com.jenkins.testforjenkins.Test01Test@4566e5bd]" name="openBaiDu" duration-ms="2045" started-at="2019-07-08T14:02:36Z" finished-at="2019-07-08T14:02:38Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [融贝网官网_值得信赖的网络借贷信息中介平台] but found [百度新闻——海量中文资讯平台]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [融贝网官网_值得信赖的网络借贷信息中介平台] but found [百度新闻——海量中文资讯平台]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.jenkins.testforjenkins.Test01Test.openBaiDu(Test01Test.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBaiDu -->
        <test-method status="PASS" signature="closeDriver()[pri:0, instance:com.jenkins.testforjenkins.Test01Test@4566e5bd]" name="closeDriver" is-config="true" duration-ms="1249" started-at="2019-07-08T14:02:38Z" finished-at="2019-07-08T14:02:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
      </class> <!-- com.jenkins.testforjenkins.Test01Test -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
